staging/src/k8s.io/kubectl/pkg/describe/describe.go

```diff
@@ -183,8 +188,14 @@ func (npw *nestedPrefixWriter) WriteLine(a ...interface{}) {
 	npw.PrefixWriter.Write(npw.indent, "%s", fmt.Sprintln(a...))
 }
 
-func describerMap(clientConfig *rest.Config) (map[schema.GroupKind]ResourceDescriber, error) {
-	c, err := clientset.NewForConfig(clientConfig)
+func describerMap(clientConfig *rest.Config, client *http.Client) (map[schema.GroupKind]ResourceDescriber, error) {
+	var c *clientset.Clientset
+	var err error
+	if client == nil {
+		c, err = clientset.NewForConfig(clientConfig)
```

Comment:
any callers that hit this code path will still spawn new transports, right? do we know if any kubectl callers hit this code path?